src/Apps/Home/Components/HomeContentCards/HomeContentCard.tsx

```diff
@@ -21,119 +21,157 @@ interface HomeContentCardProps {
   onClick?: () => void
 }
 
-export const HomeContentCard: React.FC<HomeContentCardProps> = ({
+export const HomeContentCard: React.FC<HomeContentCardProps> = props => {
+  return (
+    <Box width="100%" height="100%">
+      <Media at="xs">
+        <HomeContentCardSmall {...props} />
+      </Media>
+
+      <Media greaterThan="xs">
+        <HomeContentCardLarge {...props} />
+      </Media>
+    </Box>
+  )
+}
+
+const HomeContentCardSmall: React.FC<HomeContentCardProps> = ({
   card,
   index,
   onClick,
 }) => {
-  const extras = card.extras || {}
+  const image = card.imageUrl
+    ? cropped(card.imageUrl, {
+        // 3:2
+        width: 500,
+        height: 333,
+      })
+    : null
+
+  return (
+    <RouterLink
+      onClick={onClick}
+      display="block"
+      width="100%"
+      height="100%"
+      to={card.url!}
+      bg="black5"
+      textDecoration="none"
+      aria-label={`${card.title} - ${card.description}`}
```

Comment:
Made these so that they only have a single link which has a label now for better accessibility. Previously it would just focus the buttons.